modules/microshift-gitops-debug.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// microshift_running_apps/microshift-gitops.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-gitops-debug_{context}"]
+= Debugging GitOps on {microshift-short}
+
+There are two ways you debug GitOps applications in your {microshift-short} instance: Using sos reports 
+
+.Prerequisites 
+
+* You must have the `sos` package installed 
+
+.Procedure
+
+. Using sos reports to debug GitOps.
+
+.. Log into the host as a root user. 
+.. Perform the debug report creation procedure by running the following command:
++
+[source,terminal]
+----
+sos report 
```

Comment:
@ggiguash Hi Giga! Do you know which commands work for debugging on MicroShift?

Reply:
Yes, we have a special script we recommend running as described here: https://access.redhat.com/documentation/en-us/red_hat_build_of_microshift/4.15/html/support/microshift-sos-report#gathering-data-microshift-sos-report_microshift-sos-report

So, the command should be:
```
$ microshift-sos-report
```